app/src/androidTest/java/org/mozilla/tv/firefox/ui/TooltipTest.kt

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.ui
+
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.tv.firefox.helpers.MainActivityTestRule
+import org.mozilla.tv.firefox.ui.robots.navigationOverlay
+
+private const val TURBO_MODE_TEXT = "Turbo Mode "
+private const val BACK_TEXT = "Navigate back"
+private const val FORWARD_TEXT = "Navigate forward"
+private const val RELOAD_TEXT = "Reload website"
+private const val PIN_TEXT = "Pin to homescreen "
+private const val DESKTOP_MODE_TEXT = "Request desktop version of this site "
+private const val EXIT_TEXT = "Exit Firefox"
+private const val SETTINGS_TEXT = "Settings"
+private const val ON_TEXT = "ON"
+private const val OFF_TEXT = "OFF"
+
+/**
+ * A test for the nav bar tooltips including:
+ * - Each tooltip shows the correct string
+ * - The string dynamically updates the ON/OFF state (only applicable to Turbo Mode)
+ */
+class TooltipTest {
+
+    @get:Rule val activityTestRule = MainActivityTestRule()
+
+    /* ktlint-disable no-blank-line-before-rbrace */ // This imposes unreadable grouping.
+    @Test
+    fun tooltipTest() {
+        navigationOverlay {
+            remoteUp()
```

Comment:
Let's add a comment that by default URL bar is selected when overlay is opened, and pressing UP will select Turbo Mode. If this behavior changes in the future, then we would know why we did it this way